swagger.yaml

```diff
@@ -2332,6 +2324,108 @@ components:
+      required:
+        - taskId
+        - taskName
+      properties:
+        taskId:
+          description: ID of the task that was assigned
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that was assigned
+          type: string
+
+    TaskStatusChangesNotificationData:
+      type: object
+      description: Additional data for the TASK_STATUS_CHANGED notification
+      required:
+        - taskId
+        - status
+        - taskName
+      properties:
+        taskId:
+          description: ID of the task that was updated
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that was assigned
+          type: string
+        newStatus:
+          $ref: '#/components/schemas/TaskStatus'
+          description: New status of the task
+
+    ChangeProposalCreatedNotificationData:
+      type: object
+      description: Additional data for the CHANGE_PROPOSAL_CREATED notification
+      required:
+        - changeProposalId
+        - taskId
+        - taskName
+      properties:
+        changeProposalId:
+          description: ID of the change proposal that was created
+          type: string
+          format: uuid
+        taskId:
+          description: ID of the task that the change proposal is related to
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that the change proposal is related to
+          type: string
+
+    ChangeProposalStatusChangedNotificationData:
+      type: object
+      description: Additional data for the CHANGE_PROPOSAL_STATUS_CHANGED notification
+      required:
+        - changeProposalId
+        - taskId
+        - taskName
+        - newStatus
+      properties:
+        changeProposalId:
+          description: ID of the change proposal that was updated
+          type: string
+          format: uuid
+        taskId:
+          description: ID of the task that the change proposal is related to
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that the change proposal is related to
+          type: string
+        newStatus:
+          $ref: '#/components/schemas/ChangeProposalStatus'
+          description: New status of the change proposal
+
+    CommentLeftNotificationData:
+      type: object
+      description: Additional data for the COMMENT_LEFT notification
+      required:
+        - commentId
+        - taskId
+        - taskName
+        - comment
+      properties:
+        commentId:
+          description: ID of the comment that was left
+          type: string
+          format: uuid
+        taskId:
+          description: ID of the task that the comment is related to
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that the comment is related to
+          type: string
+        comment:
```

Comment:
Same here, trying to avoid the unnecessary requests. If we don't need this data, it can be removed.

Reply:
Currently the notifications read only "A new comment has been added to TaskName". So we could remove the commentId and comment properties. Although it seems reasonable to include them to me. So maybe we can leave these in for now.